lib/auth/MatrixAuthProvider.js

```diff
@@ -169,7 +180,8 @@ class MatrixAuthProvider {
         const medienhausMetaEvent = {
             type: type,
             template: template,
-            version: '0.4',
+            application: application,
```

Comment:
What's the difference between `template` and `application`?

At the same time, like I said above: I think we don't actually make use of this new "application" parameter anywhere. So let's remove it again?